src/lib/jitter/jitter.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+* Jitter utility functions
+* (C) 2024 Planck Security S.A.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <span>
+
+#include "jitter.h"
+
+#ifdef BOTAN_HAS_JITTER
+
+   #include <memory>
+
+namespace Botan {
+
+rand_data* jitter_collector_create() {
+   static int result = jent_entropy_init();
+
+   if(result == 0) {
+      return jent_entropy_collector_alloc(1, 0);
```

Comment:
The parameters here are unclear (and I say this even after examining the declarations and comments in `jitterentropy.h`) so I think named constants + comments are warranted.